index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Santiago's Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="styles/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    
+</head>
+<body>
+    <section class="toolbar">
+        <div>
+            <a href="#"><img src="images/logo.png"></a>
+        </div>
+        <nav class="navigation">
+            <a href="#">Portfolio</a>
+            <a href="#">About</a>
+            <a href="#">Contact</a>
+        </nav>
+        <div class="menu">
+            <a href="#"><img src="images/Union.png"></a>
+        </div>
+    </section>
+    <section class="headline">
+        <h1>I'm Santiago <br>Glad to see you!</h1>
+        <p class="resume">I'm a software developer! I can help you build a product, 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+        <h4 class="connect">LET'S CONNECT</h4>
+        <div>
+            <a class="media" href="https://twitter.com/Santiag24209785"><img src="images/twitter.png"></a>
+            <a class="media" href="https://www.linkedin.com/in/santiago-munoz-0b2b1a260/"><img src="images/linkedin.png"></a>
+            <a class="media" href="https://github.com/smunoz1988"><img src="images/github.png"></a>
+        </div>
```

Comment:
Nice icons though 💯 but to improve this further, the list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. 🔗